sql/handler.h

```diff
@@ -2021,6 +2021,11 @@ class handler :public Sql_alloc
   {
     cached_table_flags= table_flags();
   }
+  /**
+    For MyRocks, secondary initialization that happens after frm is parsed into
```

Comment:
... this one is not, and again it should mention as an example that MyRocks uses it to adjust HA_PRIMARY_KEY_IN_READ_INDEX